Games/core/Player.js

```diff
@@ -248,6 +248,14 @@ module.exports = class Player {
         return will;
     }
 
+    getVegMeeting() {

```

Comment:
there is a getmeetingbyname function

Reply:
is it possible to check this.game.vegmeeting instead?

Reply:
furthermore, this function getVegmeeting is only used to check the logic "hasVegMeeting"

would this work? i am not sure

```
hasVegMeeting() {
   return this.game.vegMeeting !== undefined
}
```

Reply:
sorry it cannot has useVegMeeting, needs to be getVegMeeting